theano/tensor/subtensor.py

```diff
@@ -40,6 +41,16 @@ def __init__(self, *args):
         TypeError.__init__(self, *args)
 
 
+class AdvancedBooleanIndexingError(TypeError):
+    """
+    Raised when Subtensor is asked to perform advanced indexing with boolean masks.
+
+    """
+
+    def __init__(self, *args):
```

Comment:
You don't need this init method. Just use "pass" as the content of this class for python syntax.